Unable to build from source at least on ArchLinux

I'm trying to update my [AUR recipe](https://aur.archlinux.org/packages/obs-rtspserver/) of obs-rtspserver but when I compile it, I get this:
``` sh
/home/tytan652/Programming/AUR/obs-rtspserver/src/obs-studio-26.1.2/plugins/obs-rtspserver-2.0.3/rtsp-server/xop/AACSource.cpp: In member function ‘virtual std::string xop::AACSource::GetAttribute()’:
/home/tytan652/Programming/AUR/obs-rtspserver/src/obs-studio-26.1.2/plugins/obs-rtspserver-2.0.3/rtsp-server/xop/AACSource.cpp:73:34: error: format not a string literal and no format arguments [-Werror=format-security]
   73 |   snprintf(nullptr, 0, rtpmap_fmt) + strlen(fmtp_fmt);
      |                                  ^
cc1plus: some warnings being treated as errors
make[2]: *** [plugins/obs-rtspserver-2.0.3/rtsp-server/CMakeFiles/rtsp-server.dir/build.make:314: plugins/obs-rtspserver-2.0.3/rtsp-server/CMakeFiles/rtsp-server.dir/xop/AACSource.cpp.o] Error 1
make[2]: Leaving directory '/home/tytan652/Programming/AUR/obs-rtspserver/src/obs-studio-26.1.2/build'
make[1]: *** [CMakeFiles/Makefile2:1705: plugins/obs-rtspserver-2.0.3/rtsp-server/CMakeFiles/rtsp-server.dir/all] Error 2
make[1]: Leaving directory '/home/tytan652/Programming/AUR/obs-rtspserver/src/obs-studio-26.1.2/build'
make: *** [Makefile:156: all] Error 2
make: Leaving directory '/home/tytan652/Programming/AUR/obs-rtspserver/src/obs-studio-26.1.2/build'
```
Note that AUR packagers should not modify default flags.
Edit: The same happen with 2.0.1 and 2.0.2. I tried to build it on another ArchLinux machine same happen

<br><br>

PS: There is two AUR recipes for obs-rtspserver:
- `obs-rtspserver` built from source
- `obs-rtspserver-bin` made with precompiled binaries

--- a/rtsp-server/xop/AACSource.cpp
+++ b/rtsp-server/xop/AACSource.cpp
@@ -72,7 +72,7 @@ string AACSource::GetAttribute() // RFC 3640
 			       "sizelength=13;indexlength=3;indexdeltalength=3;"
 			       "config=%02X%02X";
 	const size_t buf_size =
-		snprintf(nullptr, 0, rtpmap_fmt) + strlen(fmtp_fmt);
+		snprintf(nullptr, 0, rtpmap_fmt, samplerate_, channels_) + strlen(fmtp_fmt);
 	auto buf = vector<char>(buf_size);
 	const size_t rtpmap_format_size =
 		sprintf(buf.data(), rtpmap_fmt, samplerate_, channels_);

